iothub/device/src/RetryPolicies/NoRetry.cs

```diff
@@ -10,6 +11,16 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public class NoRetry : IRetryPolicy
     {
+        /// <summary>
+        /// Create an instance of a retry policy that perfrms no retries.
+        /// </summary>
+        public NoRetry()
+        {
+            if (Logging.IsEnabled)
+                Logging.Info(this, $"NOTE: A no-retry retry policy has been enabled," +
```

Comment:
you can just says 'A no-retry policy has been enabled'

Reply:
I preferred specifying what kind of policy a "no-retry" policy was; do you think it seems redundant?